src/main/java/exception/NullDescription.java

```diff
@@ -0,0 +1,14 @@
+package exception;
+
+public class NullDescription extends DukeException {
+
+    private String type;
+
+    public NullDescription(String type) {
+        this.type = type;
+    }
+    @Override

```

Comment:
I think there should be a space between lines 9 and 10?

Reply:
Thanks for the spot!